.github/policies/issues.close-duplicate-issues.yml

```diff
@@ -0,0 +1,23 @@
+name: Issue Triage
+description: Actions associated with the issues triage process
+resource: repository
+configuration:
+  resourceManagementConfiguration:
+    scheduledSearches:
+    - description: Close duplicate issues
+      frequencies:
+      - hourly:
+          hour: 3
+      filters:
+      - isIssue
+      - isOpen
+      - hasLabel:
+          label: duplicate
+      - noActivitySince:
+          days: 3
+      actions:
+      - addReply:
+          reply: This issue has been marked as duplicate and has not had any activity for **3 days**. It will be closed for housekeeping purposes.
+      - closeIssue
+onFailure: 
+onSuccess: 
```

Comment:
Wait this file is (ironically) itself a duplicate. This one has the issue**s** (plural) prefix problem as well. I think this version was meant ot be deleted